lib/matplotlib/__init__.py

```diff
@@ -1443,13 +1441,17 @@ def _add_data_doc(docstring, replace_names):
     -------
         The augmented docstring.
     """
-    docstring = inspect.cleandoc(docstring) if docstring is not None else ""
-    repl = ("* All positional and all keyword arguments."
-            if replace_names is None else
-            ""
-            if len(replace_names) == 0 else
-            "* All arguments with the following names: {}.".format(
-                ", ".join(map(repr, sorted(replace_names)))))
+    if (docstring is None
+            or replace_names is not None and len(replace_names) == 0):
+        return docstring
+    docstring = inspect.cleandoc(docstring)
+    repl = (
+        ("    each other argument can also be string *s*, which is\n"
```

Comment:
```suggestion
        ("    each other argument can also be string ``s``, which is\n"
```
It's not a parameter name, but a code element (variable or literal).

Reply:
indeed, fixed